publication.gradle

```diff
@@ -0,0 +1,69 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+group = PUBLISH_GROUP_ID
+version = PUBLISH_VERSION
+
+def prepareEnvironment() {
+    ext["ossrhUsername"] = System.getenv('OSSRH_USERNAME')
+    ext["ossrhPassword"] = System.getenv('OSSRH_PASSWORD')
+    ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID')
+    ext["signing.password"] = System.getenv('SIGNING_PASSWORD')
```

Comment:
Minor (🔍): Consider moving this below the `ext["signing.key"]` in order to match the order within the `useInMemoryPgpKeys`. As you can tell, this is purely an OCD related comment because of consistency reasons, to have the `keyId` and `key` properties grouped, and the password last... 😅